scripts/get_og_previews.py

```diff
@@ -55,9 +57,28 @@ def get_og_preview(url):
     # TODO: Maybe add additional behaviour to check if the image is large enough
     soup = get_page(url)
     image_url = get_og_image(soup)
+
     return image_url
 
 
+def write_image_to_file(url: str, name: str) -> Path | None:
```

Comment:
```suggestion
def write_image_to_file(url: str, image_id: str) -> Path | None:
```
I feel `image_id` is a bit more clear on the uniqueness requirement.